lib/stellar/client.rb

```diff
@@ -244,7 +252,7 @@ def check_memo_required(tx_envelope)
         end
         if info.data["config.memo_required"] == "MQ=="
           # MQ== is the base64 encoded string for the string "1"
-          raise AccountRequiresMemoError.new("account requires memo")
+          raise AccountRequiresMemoError.new("account requires memo", destination, idx)
```

Comment:
@JakeUrban nice! can you update the test to assert this and also update the demo code so people know what attrs they can work with?

Reply:
Sure thing

Reply:
When you say demo code you're referring to the `/examples` files right? I don't want to update the examples before the functionality is released. I think doing so would confuse people.

But I'll add that test.